ddgr

```diff
@@ -23,6 +23,7 @@ import html.entities
 import html.parser
 import logging
 import os
+import pyperclip
```

Comment:
Please make it optional (as we do with the `readline` library) and put all usage within exception handler.